src/UI/Tables/Stateful.elm

```diff
@@ -417,6 +428,13 @@ updateSelectionSet state item value =
     )
 
 
+updateExpandEllipsis : StateModel msg item columns -> Int -> Int -> ( State msg item columns, Cmd msg )
+updateExpandEllipsis state row col =
+    ( State { state | expandedEllipsis = Just ( row, col ) }
+    , Cmd.none
+    )
+
```

Comment:
I indeed tried, but the focus in one element can't trigger the transparency change in one of its sub-components.
Besides that, a shenanigan is currently required so that a simple "el" is focusable.

Reply:
Maybe we could talk to Sufiyan about leaving the button always visible (TBH I think it would be better that way because we don't hide the action from the user). The issue with the transparency could be solved using a child selector I guess.
Anyway, it's up to you in the end, but I have the feeling that this might be too complex for a somewhat ordinary ui element. There are implementations of tooltips that only use css, something on the lines of:
```css
[data-tooltip] {
  content: attr('data-tooltip');
}

[data-tooltip]:after,
[data-tooltip]:before {
  // ...
}
```